manifest.json

```diff
@@ -3,10 +3,13 @@
   "name": "buyer-org-supplier",
   "version": "0.6.0",
   "title": "Buyer Organization Supplier",
-  "description": "A app for handle buyer organizations feature for suppliers",
+  "description": "A app to handle the buyer organizations feature for suppliers",
   "builders": {
     "docs": "0.x",
-    "node": "6.x"
+    "node": "6.x",
+    "admin": "0.x",
```

Comment:
Since we are not releasing this now I would suggest to comment/remove this admin so the code keeps untouchable

Reply:
```suggestion

```